samples/zigbee/light_bulb/README.rst

```diff
@@ -11,14 +11,16 @@ This :ref:`Zigbee <ug_zigbee>` light bulb sample demonstrates a simple light bul
 
 You can use this sample with the :ref:`Zigbee network coordinator <zigbee_network_coordinator_sample>` and the :ref:`Zigbee light switch <zigbee_light_switch_sample>` to set up a basic Zigbee network.
 
+|zigbee_sample_fem_support|
```

Comment:
I removed this line from Zigbee samples and shortcuts.txt earlier today. Please fix.